evalml/tests/component_tests/test_featuretools.py

```diff
@@ -241,6 +241,31 @@ def test_dfs_with_serialized_features_dataframe_name(
     assert es_df_name == expected_dataframe_name
 
 
+@patch("evalml.pipelines.components.transformers.preprocessing.featuretools.dfs")
+@patch(
+    "evalml.pipelines.components.transformers.preprocessing.featuretools.calculate_feature_matrix",
+)
+def test_dfs_with_empty_input_features(
+    mock_calculate_feature_matrix,
+    mock_dfs,
+    X_y_binary,
+):
+    """Confirms that the features arg being an empty list is not treated the same as
+    it being unspecified."""
```

Comment:
Can we be explicit about testing this? i.e. generate two DFS objects, one with `features=[]` and one with `features=None` and test the differences?